.github/workflows/make-release.yaml

```
@@ -153,3 +153,18 @@ jobs:
           asset_path: pkg/pkg-windows-latest-oldrel/${{ env.PACKAGE_NAME }}_${{ env.PACKAGE_VERSION }}.zip
           asset_name: ${{ env.PACKAGE_NAME }}_${{ env.PACKAGE_VERSION }}_oldrel.zip
           asset_content_type: application/zip
+
+      - name: Build random library
```

```suggestion
      - name: Create archive of random library
```
Just as this is the code, not a built executable right? Might be a little clearer